vidarrtest-regression.json.in

```diff
@@ -223,7 +223,7 @@
             {
                 "metrics_calculate": "@CHECKOUT@/tests/calculate.sh",
                 "metrics_compare": "@CHECKOUT@/tests/compare.sh",
-                "output_metrics": "/.mounts/labs/gsi/testdata/delly/output_metrics/SomaticTest.metrics",
```

Comment:
Same as above, this name is useful for developing since it tells us what is the nature of the test without exposing any OICR-specific information